m4/libpcre.m4

```diff
@@ -0,0 +1,30 @@
+AC_SUBST(PCRE_LIBS)
+AC_SUBST(PCRE_CFLAGS)
+
+PKG_CHECK_MODULES(
+    [PCRE],
+    [libpcre2-8],
+    [
+        found_libpcre=yes
+        AC_DEFINE(
+            [PCRE2_CODE_UNIT_WIDTH],
+            8,
+            [Define libpcre2 unit size]
+        )
+    ],
+    [
+        found_libpcre=no
+    ]
+)
+
+# SSS_AC_EXPAND_LIB_DIR()
+AS_IF([test x"$found_libpcre" != xyes],
+    [AC_CHECK_HEADERS([pcre2.h],
+        [AC_CHECK_LIB([libpcre2-8],
```

Comment:
You should use pkgconfig here rather than checking for a specific version IMO. In pcre2-devel there should be: `/usr/lib64/pkgconfig/libpcre2-posix.pc`

Reply:
I'm not familiar with what you asking in regards to pkgconfig (looking into it though).  I just used the same approach that SSSD used when they migrated to pcre2.  So do you meaning adding this to the specfile and removing the "AS_IF" section from the m4 file?  

    BuildRequires: pkgconfig(libpcre2-8)

The only concern I have here (if that's what you meant) is that no one uses our specifle but us

Reply:
Fixed